1-exercises/B-highlight-words/script.js

```diff
@@ -1,7 +1,34 @@
 function highlightWords(paragraph, colours) {
+  
```

Comment:
This doesn't work, you're close but not finished, I think maybe some pair programming to figure it out wouldn't hurt